graphistry/Plottable.py

```diff
@@ -81,12 +93,14 @@ class Plottable(object):
     _node_features : Optional[pd.DataFrame]
     _node_features_raw: Optional[pd.DataFrame]
     _node_target : Optional[pd.DataFrame]
+    _node_target_raw : Optional[pd.DataFrame]
```

Comment:
I've also been thinking, again maybe beyond scope here:

```
_common_attr
_plugin_state: Dict[PLUGIN_TYPE, Any]
```
or some other encapsulation, so we aren't just making these massive top-level slots for everything...